ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/drawing-and-display.tsx

```diff
@@ -379,7 +411,15 @@ export const OpenlayersDrawings = ({
     )
     wreqr.vent.trigger(`search:drawend`, drawingModel)
     drawingModel.set('drawing', false)
-    drawingModel.set(convertToModel(drawingLocation, drawingShape))
+
+    // preserve buffer set by user
+    const updatedDrawingLocation = preserveAttributes(
+      drawingModel,
+      drawingLocation,
+      drawingShape
+    )
+
+    drawingModel.set(convertToModel(updatedDrawingLocation, drawingShape))
     setIsDrawing(false)
     setDrawingGeometry(drawingLocation)
```

Comment:
✏️ Recommend passing `updatedDrawingLocation` here as well. It doesn't make any functional difference, since it points to the same object, but it currently looks weird to use `updatedDrawingLocation` above, but not here.